### DISCUSSION TITLE
Like Pivot Table

### DISCUSSION THREAD

**Discussion by 12re34enes** [apache/superset#27362] [General]
![pivot](https://github.com/apache/superset/assets/155597786/bf73f52f-ec3b-4380-82e5-3a910b4ca7b9)
How can we make such a table in Superset, especially the row with 'A)', the part named 'Genel Toplam' on the far right and the row named 'Genel Toplam' at the bottom. Can you help us with this?

**Comment by michael-s-molina** [comment]
@12re34enes Please follow the issue template recommendation:

> 🙏 Asking a question or getting help: post in the [Superset Slack chat](http://bit.ly/join-superset-slack) or [GitHub Discussions](https://github.com/apache/superset/discussions) under "Q&A / Help".

I'm converting this to a GitHub discussion as it's more appropriate. It can also be a Slack thread.
